### ISSUE TITLE
Support for multisig wallets

### ISSUE THREAD

**Issue by Nathan-Huckleberry** [algorand/algorand-wallet#56]
Feature request to add multisig wallet support.

A user should be able to create a multisig wallet by specifying public keys and should be able to sign transactions with their relevant public keys.

**Comment by Nathan-Huckleberry** [comment]
Prototyping the multisig wallet creation experience for android.

![image](https://user-images.githubusercontent.com/16107760/147607622-85482f48-b937-484d-ad9f-7decca021057.png)
![image](https://user-images.githubusercontent.com/16107760/147607627-a8f8b876-8098-4fb6-9827-40bcee3d6b88.png)
![image](https://user-images.githubusercontent.com/16107760/147607631-d6032c94-53bd-4d01-81b6-d460dbf54945.png)

I don't have the hardware required to work on the ios app. It'd be nice if someone could build the same experience on ios.

**Comment by mateusz-sznura-ulam** [comment]
Hello! What's the status of this one? Is there a support for signing multisig txns using Pera Wallet already? I'm trying to sign a multisig transaction using Pera Wallet, but I'm getting this error: `Error: Invalid Input: The requested signer is not in the User Wallet`
<img width="595" alt="image" src="https://user-images.githubusercontent.com/100783811/158123026-fd993abf-93c4-4511-a20a-68960306d5f2.png">

**Comment by Nathan-Huckleberry** [comment]
No, I haven't finished this. I'll see if I can make some time later this week to finish it up. 

My code only applies to the Android app.

**Comment by mateusz-sznura-ulam** [comment]
And will it work (signing multisig transaction) if Pera Wallet is paired with Ledger?

**Comment by Nathan-Huckleberry** [comment]
Yes, I intend to support hardware wallets.

**Comment by maxcoto** [comment]
Hey Nathan! Hope you're doing great. Is it there any ETA for this feature? Let me know if you need help with code, reviews or tests. We are willing to contribute as we are in the need for multisig signing app calls with a ledger device.

**Comment by nathan-huckleberry** [comment]
The transition to pera-wallet made most of my code obsolete. Feel free to pick up this issue.

**Comment by taylanpince** [comment]
Thank you for this. multisig support is on our roadmap and we will have more updates on this later this year. Since this feature is captured on our roadmap (https://roadmap.perawallet.app/feature-requests), I am closing this issue.
